Formula/step.rb

```diff
@@ -21,7 +21,7 @@ class Step < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    contents = Dir["{*,.git,.gitignore}"]
+    contents = Dir.glob("*", File::FNM_DOTMATCH) - %w[. ..]
     (buildpath/"src/github.com/smallstep/cli").install contents
```

Comment:
@dopey How hard would it be to push this complexity to the build system? i.e., just be able to run `system "make", "build"` from the default top level build directory? If it's not too difficult, would upstream be amenable to this? I guess it's possible that this could break other package managers, so I understand if that isn't feasible.

Reply:
The project uses [`dep`](https://github.com/golang/dep) to vendor dependencies, and as far as I can tell `dep` requires that a package be under `$GOPATH/src`. I'm not super well acquainted with this stuff, but based on some searching it seems that the other workaround (other than copying the whole project into `$GOPATH/src` - what we're currently doing) would be to do some [symlink magic](https://github.com/golang/dep/issues/911) - which I think would be more confusing. Dependency mgmt is scheduled to become a first class citizen in golang soon, so once we replace `dep` the `$GOPATH/src` requirement might go away.

Like I said, I'm not super confident with this stuff. If there are examples of other Formulas that use `dep` and are able to build directly in the `buildpath` I'd be happy to take their lead.

Reply:
This seems to be the common pattern
```
  def install
    ENV["GOPATH"] = buildpath
    (buildpath/"src/github.com/ssh-vault/ssh-vault").install buildpath.children
```
so if you grep for `buildpath.children` there will be multiple examples from what I can tell.

Reply:
Thanks for the tip! Updated. Lemme know if that's better / easier to understand.